frontend/src/pages/projects/screens/detail/storage/data.ts

```diff
@@ -45,6 +45,13 @@ export const columns: SortableData<StorageTableData>[] = [
     label: 'Storage context',
     width: 25,
     sortable: false,
+    info: {
+      popover:
+        'The context indicates the purpose of the storage: general purpose, or model storage.',
```

Comment:
```suggestion
        'The context indicates the purpose of the storage: general purpose or model storage.',
```

Idk if that's like the oxford comma but it bothers me :laughing: I'll let UX experts decide it's fate